hibernate-core/src/test/java/org/hibernate/orm/test/jpa/transaction/batch/JtaWithStatementsBatchTest.java

```diff
@@ -92,16 +92,18 @@ public void testUnableToReleaseStatementMessageIsNotLogged(EntityManagerFactoryS
 						assertStatementsListIsCleared();
 						assertAllStatementsAreClosed( testBatch.createdStatements );
 					}
-					catch (Exception e) {
+					catch (Throwable t) {
```

Comment:
I wonder if it wouldn't be better to catch both Exception + AssertionError, as you did in TransactionUtil.wrapInTransaction?

Reply:
changed it, thanks @jrenaat